src/index.js

```diff
@@ -34,16 +36,18 @@ app.use(
   })
 );
 
+app.use('/api-docs', swaggerUi.serve);
+app.get('/api-docs', swaggerUi.setup(swaggerDocument));
 if (!isProduction) {
   app.use(errorhandler());
 }
 
-if (isProduction) {
-  mongoose.connect(process.env.MONGODB_URI);
-} else {
-  mongoose.connect('mongodb://localhost/conduit');
-  mongoose.set('debug', true);
-}
+// if (isProduction) {
+//   mongoose.connect(process.env.MONGODB_URI);
+// } else {
+//   mongoose.connect('mongodb://localhost/conduit');
+//   mongoose.set('debug', true);
+// }
```

Comment:
Do not leave comments. Either remove the code or fix the issue.

Reply:
I edited the code.

Reply:
https://app.swaggerhub.com/apis-docs/BeyFaith/Barefoot-Nomad/1.0.0
This is the link to the online documentation